style/runtimeTemplate.css

```diff
@@ -36,6 +49,20 @@
   color: #000000;
   display: flex;
 }
+.create-runtime-sub-message-network{
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 15px;
+  padding-left: 28px;
+  color: var(--jp-ui-font-color2);
+  width: 50%;
+  overflow: hidden;
+  word-wrap: break-word;
+  margin-top: -5px;
```

Comment:
We should not use negative values. Please check if there is a better way of handling this